<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>1200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>1170</x>
      <y>20</y>
      <width>164</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="4" column="1">
      <widget class="QDoubleSpinBox" name="vyo">
       <property name="minimum">
        <double>-50000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>50000.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Posicion Yo</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Posicion Xo</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDoubleSpinBox" name="yo">
       <property name="minimum">
        <double>-50000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>50000.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Velocidad Vyo</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Planeta 1</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QDoubleSpinBox" name="vxo">
       <property name="minimum">
        <double>-50000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>50000.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDoubleSpinBox" name="xo">
       <property name="minimum">
        <double>-50000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>50000.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Velocidad Vxo</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>0</y>
      <width>1134</width>
      <height>738</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView_2">
    <property name="geometry">
     <rect>
      <x>1160</x>
      <y>10</y>
      <width>181</width>
      <height>131</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView_3">
    <property name="geometry">
     <rect>
      <x>1160</x>
      <y>140</y>
      <width>181</width>
      <height>131</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>1170</x>
      <y>150</y>
      <width>164</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="2" column="1">
      <widget class="QDoubleSpinBox" name="yo_2">
       <property name="minimum">
        <double>-50000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>50000.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Planeta 2</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>Velocidad Vxo</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QDoubleSpinBox" name="vyo_2">
       <property name="minimum">
        <double>-50000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>50000.000000000000000</double>
       </property>
       <property name="value">
        <double>100.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDoubleSpinBox" name="xo_2">
       <property name="minimum">
        <double>-50000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>50000.000000000000000</double>
       </property>
       <property name="value">
        <double>5000.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Posicion Xo</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Velocidad Vyo</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Posicion Yo</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QDoubleSpinBox" name="vxo_2">
       <property name="minimum">
        <double>-50000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>50000.000000000000000</double>
       </property>
       <property name="value">
        <double>50.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>670</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Iniciar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>Parar</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGraphicsView" name="graphicsView_4">
    <property name="geometry">
     <rect>
      <x>1160</x>
      <y>270</y>
      <width>181</width>
      <height>131</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget_3">
    <property name="geometry">
     <rect>
      <x>1170</x>
      <y>280</y>
      <width>164</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="4" column="1">
      <widget class="QDoubleSpinBox" name="vyo_3">
       <property name="minimum">
        <double>-50000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>50000.000000000000000</double>
       </property>
       <property name="value">
        <double>-100.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>Posicion Yo</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>Posicion Xo</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDoubleSpinBox" name="yo_3">
       <property name="minimum">
        <double>-50000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>50000.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>Velocidad Vyo</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_14">
       <property name="text">
        <string>Planeta 3</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QDoubleSpinBox" name="vxo_3">
       <property name="minimum">
        <double>-50000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>50000.000000000000000</double>
       </property>
       <property name="value">
        <double>-50.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDoubleSpinBox" name="xo_3">
       <property name="minimum">
        <double>-50000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>50000.000000000000000</double>
       </property>
       <property name="value">
        <double>-5000.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_15">
       <property name="text">
        <string>Velocidad Vxo</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGraphicsView" name="graphicsView_5">
    <property name="geometry">
     <rect>
      <x>1160</x>
      <y>400</y>
      <width>181</width>
      <height>131</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget_4">
    <property name="geometry">
     <rect>
      <x>1170</x>
      <y>410</y>
      <width>164</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="4" column="1">
      <widget class="QDoubleSpinBox" name="vyo_4">
       <property name="minimum">
        <double>-50000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>50000.000000000000000</double>
       </property>
       <property name="value">
        <double>50.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_16">
       <property name="text">
        <string>Posicion Yo</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_17">
       <property name="text">
        <string>Posicion Xo</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDoubleSpinBox" name="yo_4">
       <property name="minimum">
        <double>-50000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>50000.000000000000000</double>
       </property>
       <property name="value">
        <double>5000.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_18">
       <property name="text">
        <string>Velocidad Vyo</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_19">
       <property name="text">
        <string>Planeta 4</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QDoubleSpinBox" name="vxo_4">
       <property name="minimum">
        <double>-50000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>50000.000000000000000</double>
       </property>
       <property name="value">
        <double>-100.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDoubleSpinBox" name="xo_4">
       <property name="minimum">
        <double>-50000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>50000.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_20">
       <property name="text">
        <string>Velocidad Vxo</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGraphicsView" name="graphicsView_6">
    <property name="geometry">
     <rect>
      <x>1160</x>
      <y>530</y>
      <width>181</width>
      <height>131</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget_5">
    <property name="geometry">
     <rect>
      <x>1170</x>
      <y>540</y>
      <width>164</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_5">
     <item row="1" column="1">
      <widget class="QDoubleSpinBox" name="xo_5">
       <property name="minimum">
        <double>-50000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>50000.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_22">
       <property name="text">
        <string>Posicion Xo</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_25">
       <property name="text">
        <string>Velocidad Vxo</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QDoubleSpinBox" name="vyo_5">
       <property name="minimum">
        <double>-50000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>50000.000000000000000</double>
       </property>
       <property name="value">
        <double>-50.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_23">
       <property name="text">
        <string>Velocidad Vyo</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_21">
       <property name="text">
        <string>Posicion Yo</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDoubleSpinBox" name="yo_5">
       <property name="minimum">
        <double>-50000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>50000.000000000000000</double>
       </property>
       <property name="value">
        <double>-5000.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QDoubleSpinBox" name="vxo_5">
       <property name="minimum">
        <double>-50000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>50000.000000000000000</double>
       </property>
       <property name="value">
        <double>100.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_24">
       <property name="text">
        <string>Planeta 5</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <zorder>graphicsView</zorder>
   <zorder>graphicsView_6</zorder>
   <zorder>graphicsView_2</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>graphicsView_3</zorder>
   <zorder>layoutWidget_2</zorder>
   <zorder>horizontalLayoutWidget</zorder>
   <zorder>graphicsView_4</zorder>
   <zorder>layoutWidget_3</zorder>
   <zorder>graphicsView_5</zorder>
   <zorder>layoutWidget_4</zorder>
   <zorder>layoutWidget_5</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1600</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
